src/Shift-Changes.package/ShClassChanged.class/instance/builder..st

```diff
@@ -0,0 +1,4 @@
+announcing
+builder: anObject
```

Comment:
the same implementation as in case of ShMetaclassChanged, It should be moved to ShAbstractChange as and called from subclasses